src/main/java/edu/cornell/kfs/coa/document/validation/impl/SubAccountGlobalPreRules.java

```diff
@@ -47,4 +55,34 @@ protected boolean checkOffCampus(MaintenanceDocument maintenanceDocument) {
 
         return continueRules;
       }
+
+    protected boolean anyOffCampusIndicatorsAreSet(SubAccountGlobal subAccountGlobal) {
+        if (subAccountGlobal.getA21SubAccount().isOffCampusCode() || subAccountGlobal.isNewSubAccountOffCampusCode()) {
+            return true;
+        } else {
+            return subAccountGlobal.getSubAccountGlobalNewAccountDetails().stream()
+                    .anyMatch(SubAccountGlobalNewAccountDetail::isOffCampusCode);
+        }
+    }
+
+    protected boolean checkContinuationAccountsForNewSubAccounts(MaintenanceDocument maintenanceDocument) {
+        SubAccountGlobal subAccountGlobal = (SubAccountGlobal) maintenanceDocument.getNewMaintainableObject().getBusinessObject();
+        subAccountGlobal.getSubAccountGlobalNewAccountDetails().stream()
+                .forEach(this::updateDetailToUseContinuationAccountIfNecessary);
```

Comment:
Is .stream() needed since the getSubAccountGlobalNewAccountDetails() returns a List?

Reply:
Good point, the list also has a forEach() method so I don't need to explicitly create the stream first. I'll update this line accordingly.